locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -409,7 +409,7 @@ protected CloudMachineNamer getCloudMachineNamer(ConfigBag config) {
         }
     }
 
-    protected Collection<JcloudsLocationCustomizer> getCustomizers(ConfigBag setup) {
+    public Collection<JcloudsLocationCustomizer> getCustomizers(ConfigBag setup) {
```

Comment:
I'm slightly uncomfortable about this becoming public - but I see why you had to do it, to let it be called by `CloudExplorerSupport.java`.